translation/src/test/java/com/google/cloud/trace/zipkin/translation/TraceTranslatorTest.java

```diff
@@ -112,27 +73,51 @@ public void testTranslateTrace() {
 
   @Test
   public void testMultihostServerRootSpan() {
-    Span span1 = Span.builder().traceId(1).id(1).name("/a")
-        .timestamp(1474488796000000L) // This is set because the server owns the span
-        .duration(5000000L)
-        .addAnnotation(Annotation.create(1474488796000000L, SERVER_RECV, serverEp))
-        .addAnnotation(Annotation.create(1474488801000000L, SERVER_SEND, serverEp))
-        .build();
-    Span span2 = Span.builder().traceId(1).parentId(1L).id(2).name("/b?client")
-        .timestamp(1474488797000000L) // This is set because the client owns the span.
-        .duration(1500000L)
-        .addAnnotation(Annotation.create(1474488797000000L, CLIENT_SEND, clientEp))
-        .addAnnotation(Annotation.create(1474488798500000L, CLIENT_RECV, clientEp))
-        .build();
-    Span span3 = Span.builder().traceId(1).parentId(1L).id(2).name("/b?server")
-        // timestamp is not set because the server does not own this span.
-        .addAnnotation(Annotation.create(1474488797500000L, SERVER_RECV, serverEp))
-        .addAnnotation(Annotation.create(1474488798300000L, SERVER_SEND, serverEp))
-        .build();
-    Span span4 = Span.builder().traceId(1).parentId(2L).id(3).name("custom-span")
-        .timestamp(1474488797600000L)
-        .duration(200000L)
-        .build();
+    Span span1 =
+        Span.newBuilder()
+            // TODO(anuraag): This seems like it should have KIND == SERVER, but that causes the
```

Comment:
Please check this note that is left over, I'm not sure if my reasoning is wrong, or the parent ID rewriting logic is wrong.

Reply:
definitely if a span ends up with "sr" "ss" something is wrong :)

Reply:
Ah ok - I went ahead and removed all the sr/cr type annotations from the tests to make that clearer.